node/utils/Queue.ts

```diff
@@ -488,11 +499,7 @@ export const processUncompleted = async (
     }
   }
 
-  if (
-    uncompletedStep !== -1 &&
-    uncompletedStep !== null &&
-    uncompletedStep !== undefined
-  ) {
+  if (uncompletedStep !== -1 && !uncompletedStep) {
```

Comment:
```suggestion
  if (uncompletedStep && uncompletedStep !== -1) {
```
I think this is the logic you want: `uncompletedStep` is truthy (not `null` or `undefined`) and it's not equal to `-1`